### ISSUE TITLE
Predictions are all _BLANK characters 

### ISSUE THREAD

**Issue by guilk** [SeanNaren/warp-ctc#63]
I am training crnn.pytorch (https://github.com/meijieru/crnn.pytorch) to recognize handwritten characters. The training loss does not decrease and the predictions are all blank characters. The warpctc passed all the tests. Also, I double checked the input images and labels. Does anyone have this issue before? Thanks. I am using Pytorch 0.4.0 and the current version of warp-ctc.

**Comment by vaibhav0195** [comment]
hi how did you solve the issue , of predicting the blank characters ?
